controllers/app.controller.js

```diff
@@ -9,3 +13,20 @@ exports.getTopics = (req, res) => {
       console.log(err);
     });
 };
+
+exports.getArticle = (req, res, next) => {
+  const { article_id } = req.params;
+  selectArticle(article_id)
+    .then((result) => {
+      res.status(200).send({ result });
+    })
+    .catch(next);
```

Comment:
unless we're throwing an error that we need to handle ourselves, we don't a catch block